2021-06-18 04:04:59,252 INFO    MainThread:4042230 [internal.py:wandb_internal():91] W&B internal server running at pid: 4042230, started at: 2021-06-18 04:04:59.252412
2021-06-18 04:04:59,259 INFO    WriterThread:4042230 [datastore.py:open_for_write():80] open: /home/chengzilong/DDQN/wandb/offline-run-20210618_040457-38ct4djk/run-38ct4djk.wandb
2021-06-18 04:04:59,259 DEBUG   HandlerThread:4042230 [handler.py:handle_request():124] handle_request: run_start
2021-06-18 04:05:01,524 DEBUG   HandlerThread:4042230 [meta.py:__init__():39] meta init
2021-06-18 04:05:01,525 DEBUG   HandlerThread:4042230 [meta.py:__init__():53] meta init done
2021-06-18 04:05:01,525 DEBUG   HandlerThread:4042230 [meta.py:probe():210] probe
2021-06-18 04:05:01,532 DEBUG   HandlerThread:4042230 [git.py:repo():33] git repository is invalid
2021-06-18 04:05:01,532 DEBUG   HandlerThread:4042230 [meta.py:_save_pip():57] save pip
2021-06-18 04:05:01,534 DEBUG   HandlerThread:4042230 [meta.py:_save_pip():71] save pip done
2021-06-18 04:05:01,534 DEBUG   HandlerThread:4042230 [meta.py:_save_conda():78] save conda
2021-06-18 04:05:05,687 DEBUG   HandlerThread:4042230 [meta.py:_save_conda():86] save conda done
2021-06-18 04:05:05,688 DEBUG   HandlerThread:4042230 [meta.py:probe():252] probe done
